src/manager.hpp

```diff
@@ -183,6 +206,38 @@ class SessionManager final :
      *        and cleanup sessions of an unavailable service.
      */
     void checkSessionOwnerAlive(const boost::system::error_code&);
+
+    /**
+     * @brief isAllSessionsAllowed check if all sessions are allowed by
+     * specified ID.
+     *
+     * @param sessionId     - unique session ID to remove from storage
+     *
+     * @return bool true/false - is closing all sessions allowed
+     */
+    bool isAllSessionsAllowed(std::string sessionId);
+    /**
+     * @brief isOwnSession check if the user is the owner of the given sessopn.
+     *
+     * @param callerSessionId     - unique session ID which request to terminate
+     * some other session
```

Comment:
Same as above

     * @param callerSessionId     - unique session ID which request to terminate
     *                              some other session